src/components/screens/PointOfInterest.js

```diff
@@ -19,12 +20,14 @@ import { TMBNotice } from '../TMB/Notice';
 import { Wrapper, WrapperWithOrientation } from '../Wrapper';
 import { OpeningTimesCard } from './OpeningTimesCard';
 import { PriceCard } from './PriceCard';
+import { getQuery, QUERY_TYPES } from '../../queries';
+import { CrossDataSection } from '../CrossDataSection';
 
 const { MATOMO_TRACKING } = consts;
 
 /* eslint-disable complexity */
 /* NOTE: we need to check a lot for presence, so this is that complex */
-export const PointOfInterest = ({ data, hideMap, navigation }) => {
+export const PointOfInterest = ({ data, hideMap, navigation, fetchPolicy }) => {
```

Comment:
passing in an external fetchpolicy, which is dependent on the refresh time for another query, does not seem like the right way to go here. maybe we should instead get a new refreshtime internally depending on the id and crossDataType?

Reply:
its data for the point of interest, so i thought it can be a proper way to use the fetch policy for the corresponding point of interest. so so the cross data refresh with the point of interest. but i can see also cases, where the point of interest should not update while the events, news, ... needs a refresh. 

will see if there is a time to add an own refresh time and fetch policy here. if you would find the time, you could add that here as well.

Reply:
added fetchpolicy for the crossdata itself, but it is located somewhere else now anyways.